index.html

```diff
@@ -0,0 +1,207 @@
+              <img class="imgP2" src="" alt="">
+              <div class="flex1">
+                     <div class="T1P2 text1"></div>
+                     <div class="T2P2 text2"></div>
+                     <div class="dashspeaker"></div>
+                     <div class="T3P2 text3"></div>
+              </div>
+       </div>
+<!-------------------------------------------------- PLACE FOR Speaker by Dynamic page- when we click on mmore button------------------------------- -->
+       <div class="speaker-person3">
+              <img class="imgP3" src="" alt="">
+              <div class="flex1">
+                     <div class="T1P3 text1"></div>
+                     <div class="T2P3 text2"></div>
+                     <div class="dashspeaker"></div>
+                     <div class="T3P3 text3"></div>
+              </div>
+       </div>
+
+       <div class="speaker-person4">
+              <img class="imgP4" src="" alt="">
+              <div class="flex1">
+                     <div class="T1P4 text1"></div>
+                     <div class="T2P4 text2"></div>
+                     <div class="dashspeaker"></div>
+                     <div class="T3P4 text3"></div>
+              </div>
+       </div>
+
+       <div class="speaker-person5">
+              <img class="imgP5" src="" alt="">
+              <div class="flex1">
+                     <div class="T1P5 text1"></div>
+                     <div class="T2P5 text2"></div>
+                     <div class="dashspeaker"></div>
+                     <div class="T3P5 text3"></div>
+              </div>
+       </div>
+
+       <div class="speaker-person6">
+              <img class="imgP6" src="" alt="">
+              <div class="flex1">
+                     <div class="T1P6 text1"></div>
+                     <div class="T2P6 text2"></div>
+                     <div class="dashspeaker"></div>
+                     <div class="T3P6 text3"></div>
+              </div>
+       </div>   
+
+</div>
+
+<div class="nonebnt">
+       <button class="bnt-more"  onclick="showSpeaker3()"  type="submit">more</button>
+       <button class="bnt-moin" onclick="hideSpeaker3()" type="submit">moin</button>
+</div>
+
+
+
+<div class="footer" >
+       <div class="footer1">
+              <div class="footer11" >Partner</div>
+              <div class="footer12" ></div>
+              <div class="footer13" >
+                     <li>Zoum-Multi-Service</li>
+                     <li>GMTECH</li>
+                     <li>INFOGUIT</li>
+                     <li>ZLK</li>
+                     <li>PPHGUI</li>
+              </div>
+       </div>
+       <div class="footer2">
+              <div class="footer21"> <div class="logofooter"><a href=""> <img src="images/icon.png " width="100%" height="100%" alt=""></a></div></div>
+              <div class="footer22"> Seminar for development in the field of national education through the application of artificial intelligence</div>
+       </div>
+</div>
+
```

Comment:
As suggested by the previous reviewer and also as stated by the project requirement, kindly align the logo to the left and the text to the right. 
I think you should be able to achieve that by using this style...
```
display: flex;
  justify-content: space-around;
  align-items: center;
```